pkg/controller/kieapp/defaults/defaults.go

```diff
@@ -1073,6 +1077,19 @@ func getKafkaConfig(kafka *api.KafkaExtObject) {
 	}
 }
 
+func getKafkaJBPMConfig(kafkaExt *api.KafkaExtObject, kafka *api.KafkaJBPMObject) {
+	// The emitter is enabled only if the kafka extension is enabled
+	if kafkaExt != nil && kafka != nil {
+		//if something is missing we set mandatory defaults
```

Comment:
thinking if this is really needed, because system properties already have predefined default values... on the other hand users can see in the KieApp (in applied) what exactly is set in their env. Our defaults are same as defaults set in system properties - see [docs](https://access.redhat.com/documentation/en-us/red_hat_process_automation_manager/7.10/html-single/integrating_red_hat_process_automation_manager_with_other_products_and_components/index#kieserver-kafka-emit-proc_integrating-amq-streams)
@spolti wdyt?

Reply:
if kieserver is setting defaults already, I agree to not set them here, just add in the field description what are the default values if empty.